com/course/zenkouv/src/Lecture3/Task1/Knight.java

```diff
@@ -37,13 +35,21 @@ public void addOutfit(Outfit outfit) {
         }
     }
 
+    public void removeOutfit(Outfit outfit) {
+        try {
+            outfits.remove(outfitsCost);
```

Comment:
you try to delete  Outfit by index == outfitsCost?? and you don't use `outfit ` param 
please do not do such crazy nonlogical thinks

Reply:
It's mistake by autocomplete.fix